### PR TITLE
[release-4.8] OCPBUGS-1455: show Limit exceeded state for large number of nodes in topology

### PR DESCRIPTION
**Pull request by sahil143**
Cherry-pick for https://github.com/openshift/console/pull/11973

### CODE REVIEW

File: frontend/packages/topology/src/components/page/LimitExceededState.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      <EmptyStateIcon variant="container" component={TopologyIcon} />
+      <Title headingLevel="h4" size="lg">
+        {t(`topology~Loading is taking longer than expected`)}
+      </Title>
+      <EmptyStateBody>
+        {t(
+          `topology~We noticed that it is taking a long time to visualize your application Topology. You can use Search to find specific resources or click Continue to keep waiting.`,
+        )}
+      </EmptyStateBody>
+      <Button variant="primary" component={(props) => <Link {...props} to="/search-page" />}>
```
Comment (vikram-raj) [inline frontend/packages/topology/src/components/page/LimitExceededState.tsx:31]: `Go to search` button redirect to the 404 page. Look like in 4.8 `/search-page` URL does not exist. We have `/search` in 4.8 for the search page.
Peek 2022-09-19 20-33

**Review by vikram-raj — COMMENTED** [review]
image

File: frontend/packages/topology/src/components/page/LimitExceededState.tsx (hunk 1 above)
Comment (sahil143) [reply frontend/packages/topology/src/components/page/LimitExceededState.tsx]: fixed

**Comment by vikram-raj** [comment]
Verified it on the 4.8 cluster bit and it works as expected.
Peek 2022-09-21 14-46

/lgtm

**Pull request merged**